### DISCUSSION TITLE
Question about small object and penetration

### DISCUSSION THREAD

**Discussion by chao0716** [bulletphysics/bullet3#3709] [General]
Hi, I am new to pybullet. I am trying to use  Franka Emika Panda at panfranka_panda/panda.urdf to grasp a small object (1cm*2cm*2cm). modified from pybullet_data/cube_small

When the object size is (3cm*5cm*5cm), the system looks fine. But when I decrease the object size to 1cm*2cm*2cm. The object becomes very easy to jump out of the gripper, and sometimes the gripper and the object penetration。

Is pybullet not supporting physical simulation with such small objects? or is there any way to improve it? 

Some people say to adjust the timestep, but the tutorial says that it should not be changed at will. 

The system looks like:
![捕获](https://user-images.githubusercontent.com/32490390/88904481-b65a2380-d287-11ea-9347-16b1300a5300.PNG)

The urdf file of the object:
`<?xml version="0.0" ?>
<robot name="cube.urdf">
  <link name="baseLink">
    <contact>
      <lateral_friction value="1"/>
      <inertia_scaling value="1.0"/>
    
    </contact>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
       <mass value="0.1"/>
       <inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="0"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
				<mesh filename="cube.obj" scale=".005 .04 .04"/>
      </geometry>
       <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
	 	<box size=".005 .04 .04"/>
      </geometry>
    </collision>
  </link>
</robot>
`

**Comment by olliejday** [comment]
I've got the same issue
